admin_ui_support/src/Controller/ConfigEntityResource.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\admin_ui_support\Controller;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\jsonapi\Controller\EntityResource;
+use Drupal\jsonapi\Exception\EntityAccessDeniedHttpException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\ConflictHttpException;
+
+/**
+ * Process Config Entity Requests.
+ */
+class ConfigEntityResource extends EntityResource {
+
+  /**
+   * {@inheritdoc}
+   *
+   * Overridden to remove validation which isn't supported by config entitie.
+   *
+   * @see https://www.drupal.org/project/drupal/issues/2300677
+   */
+  public function createIndividual(EntityInterface $entity, Request $request) {
+    $entity_access = $entity->access('create', NULL, TRUE);
```

Comment:
I'm curious, what happens on the update case? Does this just work?

Reply:
Yes works because when patching an entity `\Drupal\jsonapi\Controller\EntityResource::updateEntityField` first checks if the field is a `ContentEntityInterface` before checking field access whereas `\Drupal\jsonapi\Controller\EntityResource::createIndividual` just checks all fields for access.

This probably an oversight so since from my understanding it is not the intention of Jsonapi to support POST/PATCH on config entities. So we would probably have to expand `\Drupal\admin_ui_support\Controller\ConfigEntityResource` at some point to override `::createIndividual()`. 

Also we may have to alter the routes to produce because I am not sure why jsonapi even supports POST/PATCH methods for config entities.

Reply:
Fair, but isn't there also ```\Drupal\jsonapi\Controller\EntityResource::patchIndividual``` which should be the code path triggered here?

Reply:
Yes but `\Drupal\jsonapi\Controller\EntityResource::patchIndividual` calls which is where the actual field access check happens.